error-prone-contrib/src/test/resources/tech/picnic/errorprone/refasterrules/StreamRulesTestInput.java

```diff
@@ -90,7 +91,8 @@ ImmutableSet<Boolean> testStreamIsNotEmpty() {
   ImmutableSet<Optional<String>> testStreamMin() {
     return ImmutableSet.of(
         Stream.of("foo").max(comparingInt(String::length).reversed()),
-        Stream.of("bar").sorted(comparingInt(String::length)).findFirst());
+        Stream.of("bar").sorted(comparingInt(String::length)).findFirst(),
+        Stream.of("baz").collect(java.util.stream.Collectors.minBy(comparingInt(String::length))));
```

Comment:
Not sure about this...
For some reason, test output expcts to have the same imports are input test 😕.

While running `fmt:format` removes the expected imported packages.